src/tests/classes/token.test.ts

```diff
@@ -0,0 +1,48 @@
+import { Token, createToken, calculateDiscount } from '../../classes/token';
+
+describe('Tests of Tokens functions', () => {
+    describe('Testing the instance of Token generation', () => {
+      test('should be able to return a new Token instance', () => {
```

Comment:
it

Reply:
Não usar o test, pois o it vai ser combado com o should

"Isso deve..."
"It should..."